benchmarks/benchmark/tools/locust-load-inference/README.md

```diff
@@ -110,9 +110,10 @@ Fill out your `terraform.tfvars` with the desired model and server configuration
 - `credentials_config` - credentials for cluster to deploy Locust benchmark tool on
 - `project_id` - project id for enabling dependent services for building locust artifacts
 - `artifact_registry` - artifact registry to upload locust artifacts to
-- `inference_server_service` - an accessible service name for inference workload to be benchmarked
+- `inference_server_service` - an accessible service name for inference workload to be benchmarked **(Note: a non-80 port should be specified here. Example: `my-service-name:9000`)
```

Comment:
Is there a reason to specify a non-80 port here? I think the other ones are creating services which are exposed in port 80 by default.